unit_testing/cli/test_cli_sct_register_to_template.py

```diff
@@ -62,3 +56,49 @@ def test_sct_register_to_template_non_rpi_data(tmp_path, template_lpi):
     img_reg = Image('template2anat.nii.gz')
     # Check if both images almost overlap. If they are right-left flipped, distance should be above threshold
     assert np.linalg.norm(img_orig.data - img_reg.data) < 1
+
+
+@pytest.mark.sct_testing
+@pytest.mark.usefixtures("run_in_sct_testing_data_dir")
+@pytest.mark.parametrize("fname_gt, remaining_args", [
+    ('template/template/PAM50_small_cord.nii.gz',
+     ['-l', 't2/labels.nii.gz', '-t', 'template', '-qc', 'qc-testing', '-param',
+      'step=1,type=seg,algo=centermassrot,metric=MeanSquares:step=2,type=seg,algo=bsplinesyn,iter=5,metric=MeanSquares']),
+    (os.path.join(__sct_dir__, 'data/PAM50/template/PAM50_cord.nii.gz'),
```

Comment:
`__sct_dir__` should be taken care of by `run_in_sct_testing_data_dir`?

In https://github.com/neuropoly/spinalcordtoolbox/issues/2669#issuecomment-665153804 et al I want to get rid of `__sct_dir__`.

Also what's `template/template`? Have I just not seen that before? Why isn't that's `PAM50/template/`?

Reply:
> `__sct_dir__` should be taken care of by `run_in_sct_testing_data_dir`?

In this case, we use `__sct_dir__` to access the PAM50 template stored in the `data/` folder of SCT's installation directory (i.e. it's separate from `sct_testing_data`).

> Also what's `template/template`? Have I just not seen that before? Why isn't that's `PAM50/template/`?

This is a great question, and it caught me off guard! It looks like this is an `sct_testing_data`-specific thing: https://github.com/spinalcordtoolbox/sct_testing_data/tree/master/template

So, it looks like earlier part of your comment should be using `template/template` instead of `os.path.join(__sct_dir__, 'data/PAM50/template/PAM50_cord.nii.gz').

Reply:
I don't know what to do with this. We can think about this later.